packages/persona/src/Persona.js

```diff
@@ -5,7 +5,7 @@ Persona.ApiClient.instance.basePath = process.env.PERSONA_URL;
 
 // https://visionmedia.github.io/superagent/#timeouts
 Persona.ApiClient.instance.timeout = {
-  response: 5000, // the server only has 5 seconds to respond
+  response: 50000, // the server only has 5 seconds to respond
```

Comment:
I think this wasn't supposed to be committed 😄

Reply:
Nice catch :D Indeed it wasn't